tests/unit/models/package-info-cache/package-info-cache-test.js

```diff
@@ -466,7 +466,11 @@ describe('models/package-info-cache/package-info-cache-test.js', function () {
 
       let emberCliRealPath = path.resolve(`${projectPackageInfo.realPath}/../../../../`);
       expect(emberCliStringUtilsPkgInfo.realPath).to.equal(
-        path.join(emberCliRealPath, 'node_modules', 'ember-cli-string-utils')
+        path.join(
+          emberCliRealPath,
+          // TODO: Make this package-manager agnostic?
```

Comment:
Need some advice here I think.

Reply:
Not sure -- this is the stuff I wasn't sure would work with pnpm. Probably fine for now until it gets annoying.